style.css

```diff
@@ -60,10 +68,53 @@ li {
   color: #6070ff;
 }
 
-.navbar ul {
+.desktop-nav {
   display: none;
 }
 
+.mobile-menu {
+  position: absolute;
+  height: 750px;
+  width: 100%;
+  top: 0;
+  bottom: 0;
+  visibility: hidden;
+  background: #6070ff;
+  backdrop-filter: blur(7px);
+  mix-blend-mode: multiply;
+}
+
+.mobile-menu ul {
+  margin-top: 60px;
+}
+
+.social-items li {
+  margin-right: 15px;
+  color: #505f79;
+  transition: border-bottom 1s ease-in-out 0.2s;
+}
+
+.mobile-menu ul li {
+  line-height: 64px;
+}
+
+.toggle-btn {
+  height: 12px;
+  width: 12px;
+  font-size: 20px;
+  float: right;
+  margin-right: 38px;
+  color: #fff;
+  padding: 10px;
+  background: #6070ff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: none;
+  margin-top: 50px;
+  font-weight: bold;
+}
+
```

Comment:
- [ ] The mobile menu should be full screen in all screens starting from the smallest screen (320px of width), and also when we scroll down the mobile menu shouldn't be hidden on scroll down, it should stick in the screen until we click or tap one menu, kindly watch the following screencast for more context

https://user-images.githubusercontent.com/56886064/145872933-df1cd6bd-7221-4fef-b1aa-b44cbe849b90.mp4